ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -231,7 +261,6 @@ public class BottomSheetViewController: UIViewController {
 
     private struct Constants {
         static let animationDuration: TimeInterval = 0.25
-        static let collapsedHeight: CGFloat = 200
```

Comment:
we can still have the constant for the default collapsehieght right?

Reply:
we can keep but we are setting default value for public variable to 200 & we should not update the height collapsedHeight of constant struct. So it will be unused for us when user will update the collapsedHeight.

Reply:
I don't understand your comment     @objc public var collapsedHeight: CGFloat = Constants.collapsedHeight would still work until client decides to change it